Sources/TensorFlow/Optimizer.swift

```diff
@@ -355,3 +355,98 @@ public class AdaGrad<Model: Layer>: Optimizer
         update(&model.allDifferentiableVariables, along: direction)
     }
 }
+
+/// Adadelta Optimizer
+///
+/// It is a method that uses the magnitude of recent gradients 
```

Comment:
API documentation should fit to size (i.e. to 100 columns).

I'd suggest changing the summary to the following, similar to the Keras documentation.
> ADADELTA is a more robust extension of AdaGrad that adapts learning rates based on a moving window of gradient updates, instead of accumulating all past gradients. This way, Adadelta continues learning even when many updates have been done. Compared to AdaGrad, in the original version of ADADELTA you don't have to set an initial learning rate. In this version, initial learning rate and decay factor can be set, as in most other optimizers.